src/views/FAQPage/FAQList.vue

```diff
@@ -30,7 +30,7 @@ export default {
   },
 
   created() {
-    fetch('https://api-dev.coding.garden/faq')
+    fetch('https://api-dev.codinggarden.community/faq')
```

Comment:
If we are renaming `/faq` to `/faqs`, this needs to change but only after other "rename" related PR merges.